haystack/utils.py

```diff
@@ -245,3 +249,23 @@ def get_batches_from_generator(iterable, n):
     while x:
         yield x
         x = tuple(islice(it, n))
+
+
+def is_faiss_available() -> bool:
```

Comment:
Why do we need the methods here and are not just using `faiss_available` directly in document_store/init.py?

Reply:
In case we get rid of methods, we should rename `_faiss_available` to `FAISS_AVAILABLE`

Reply:
Yeah, You can Constant as a check.